modelci/hub/client/README.md

```diff
@@ -68,11 +68,11 @@ test_img_bytes = ...
 # init clients for different serving platforms, you can custom a client by implementing the BaseModelInspector class.
 tfs_client = CVTFSClient(test_img_bytes, batch_num=100, batch_size=32, asynchronous=False)
 # get the model info from model manager.
-mode_info = retrieve_model(architecture_name='ResNet50', framework=Framework.PYTORCH, engine=Engine.TORCHSCRIPT)
+mode_info = retrieve_model(architecture_name='ResNet50', framework=Framework.TENSORFLOW, engine=Engine.TFS)
 # init a profiler.
-profiler = Profiler(model_info=mode_info, server_name='tfs', inspector=tfs_client)
+profiler = Profiler(model_bo=mode_info, server_name='tfs', inspector=tfs_client)
 # start profiling.
-profiler.diagnose('device name')
+profiler.diagnose('cuda:0')
```

Comment:
I think @YuanmingLeee he just passing the cuda id so that in later we can use the GPUtil to get the brand name. This looks good to me since in the future there may be serving in CPU devices.

Reply:
Yep, GPUtil will get the device name and UUID via device string: e.g. `cuda:0`, `cpu`.